Assets/Scripts/Map/MapGen/MeshGenerator.cs

```diff
@@ -82,9 +82,9 @@ internal void ClearMesh()
         {
             _squareGrid2D = null;
             _squareGrid3D = null;
-            Destroy(InnerWalls3D.gameObject.GetComponent<MeshCollider>());
+            //Destroy(InnerWalls3D.gameObject.GetComponent<MeshCollider>());
```

Comment:
Why are these commented out now?

Reply:
They use 3D physics which i disabled. It should be fine to keep if we want to reenable it.